code/modules/projectiles/guns/energy/special.dm

```diff
@@ -84,8 +84,8 @@
 	ammo_x_offset = 2
 
 /obj/item/gun/energy/kinetic_accelerator/crossbow
-	name = "mini energy crossbow"
-	desc = "A weapon favored by syndicate stealth specialists."
+	name = "foam force crossbow" //SKYRAT EDIT, was mini energy crossbow
```

Comment:
Please not this one. The rest are fine. Do not make toys annoying.

Reply:
This is the intended use of the mechanic, and the crossbow is a stealthy weapon and makes the most sense to be disguised.

Reply:
So does Donksoft make toys that look like syndicate equipment (it's this one) or do syndies make equipment that looks like Donksoft toys?

At least talk with the other dude and come up with a consistent story because you both took this in different directions and I have to reiterate that I hate the idea of making toys annoying instead of fun.

Reply:
Considering DonkCo _is_ a Syndicate organization, it should be obvious that they can make toys that look like the real thing.

Reply:
Oh I see what you mean. Hm.

Reply:
Nah, I'll leave it and wait for a maintainer opinion.

Reply:
I feel like the guns by default should not look like their syndicate counterparts , but you should allow people to use wirecutters on them to remove what would be the orange barrel that makes it obvious they are toy guns to turn them into real looking ones

Reply:
I think this is actually acceptable here.

Reply:
Since it uses the TOY variant.

Reply:
It can be updated later if it causes issues.